src/renderers/dom/shared/DOMProperty.js

```diff
@@ -16,6 +16,7 @@ var invariant = require('fbjs/lib/invariant');
 // These attributes should be all lowercase to allow for
 // case insensitive checks
 var RESERVED_PROPS = {
+  aria: true,
   children: true,
   dangerouslysetinnerhtml: true,
   autofocus: true,
```

Comment:
I can't comment on the unchanged line but `onfocusin` and `onfocusout` can be removed since they're now covered by the `on` prefix rule.